src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java

```diff
@@ -75,23 +78,20 @@ public class NotificationWebSocketHandler implements WebSocketHandler {
     static final String HEADER_INDEXATION_STATUS = "indexation_status";
 
     static final String USERS_METER_NAME = "app.users";
-    static final String CONNECTIONS_METER_NAME = "app.connections";
+    static final String USER_TAG = "user";
```

Comment:
user**s** ?

Reply:
This identify each user, see the /actuator/prometheus endpoint so I think we should stick to "user":
```
# HELP app_users Number of connections per user
# TYPE app_users gauge
app_users{user="user1",} 1.0
app_users{user="user2",} 1.0
```
It's also more logical when used in grafana, we use `sum by (user) (app_users)`. Plural form does not make sense here.